class/register.js

```diff
@@ -0,0 +1,27 @@
+  // Check validation for name, mail, userId and password errors.
+  function validate() {
```

Comment:
This function could have been merged with the validate() function of forgotPass.js